components/AvalancheForecastZoneMap.tsx

```diff
@@ -174,9 +174,9 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
             mt={8}
             px={4}
             flex={1}
-            onLayout={(event: LayoutChangeEvent) => {
+            onLayout={() => {
               // onLayout returns position relative to parent - we need position relative to screen
-              event.currentTarget.measureInWindow((x, y, width, height) => {
+              topElements.current?.measureInWindow((x, y, width, height) => {
```

Comment:
Are we guaranteed to have the most up-to-date reference in `topElements.current` when the `onLayout` fires?

Reply:
The deal with component refs is that it's either the value, or it hasn't been set yet (in which case `measureInWindow` just doesn't get invoked)